lib/transformers/govukSelect.js

```diff
@@ -25,14 +25,22 @@ module.exports = ({schemaKey, schema, options, data, schemaErrors} = {}) => {
             const values = data[schemaKey];
 
             // If this item's value is included in the values, set it to selected
-            if (values.includes(item.value)) {
+            if (values === item.value) {
```

Comment:
This has been missed in the past as there is no test in for it. Can you add one to the "Pre-populate input values with supplied data" section of the tests?